app/code/Magento/Downloadable/view/adminhtml/templates/sales/items/column/downloadable/creditmemo/name.phtml

```diff
@@ -6,6 +6,7 @@
 
 /**
  * @var \Magento\Downloadable\Block\Adminhtml\Sales\Items\Column\Downloadable\Name $block
+ * @var \Magento\Framework\Escaper $escaper
  * @var \Magento\Framework\View\Helper\SecureHtmlRenderer $secureRenderer
  */
 ?>
```

Comment:
Please remove not needed closing and opening PHP tags, and combine all phpdoc blocks to a single one

Reply:
fixed